spec/dummy/config/initializers/devise.rb

```diff
@@ -0,0 +1,318 @@
+# frozen_string_literal: true
+
+# Assuming you have not yet modified this file, each configuration option below
+# is set to its default value. Note that some are commented out while others
+# are not: uncommented lines are intended to protect your configuration from
+# breaking changes in upgrades (i.e., in the event that future versions of
+# Devise change the default values for those options).
+#
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
```

Comment:
You can clean this config file, by deleting all comments

Reply:
It is a dummy project, we use it only for testing purpose.